buster/examples/gradio_app.py

```diff
@@ -27,16 +27,20 @@ def format_sources(matched_documents: pd.DataFrame) -> str:
     if len(matched_documents) == 0:
         return ""
 
-    sourced_answer_template: str = (
-        """📝 Here are the sources I used to answer your question:<br>""" """{sources}<br><br>""" """{footnote}"""
+    documents_answer_template: str = (
+        """📝 Here are the sources I used to answer your question:<br>""" """{documents}<br><br>""" """{footnote}"""
```

Comment:
Didn't notice before but the formatting is weird here, why all the """ ?

Reply:
looks like it was left overs from a previous refactor, updated to be a more sane single string